pandas/tests/computation/test_eval.py

```diff
@@ -1865,6 +1865,16 @@ def test_invalid_engine():
         pd.eval("x + y", local_dict={"x": 1, "y": 2}, engine="asdf")
 
 
+@td.skip_if_no_ne
```

Comment:
can you parameterize and text for both True/False here.

Reply:
added